generate-csv.js

```diff
@@ -58,6 +60,23 @@ function generateCSV() {
     })
 }
 
+function insertRow(rows, row) {
+    let newRow = {};
+    newRow.receiver = row.receiver
+    newRow.amount = row.amount
+    newRow.name = ''
+    newRow.token_type = 'erc20'
+    newRow.token_address = '0x333A4823466879eeF910A04D473505da62142069'
+
+    rows.push(newRow)
+}
+
+function pruneRows(rows) {
+    const floor = 0.002;
```

Comment:
Can you change this to zero?  (And then we can handle the [cut-off](https://github.com/nation3/nationcred-instance/issues/68)  in a separate pull request.)

Reply:
Done!